packages/docs/content/docs/concepts/deployment/motia-cloud.mdx

```diff
@@ -26,29 +26,52 @@ This approach provides several benefits:
 The simplest way to deploy your Motia workflows is using the CLI command:
 
 ```bash
-motia deploy --api-key <api-key> --environment <environment> --version <version>
+motia cloud deploy --api-key <api-key> --version-name <version> [options]
 ```
 
 ### Command Options
 
-| Option          | Alias | Description                           | Default           |
-| --------------- | ----- | ------------------------------------- | ----------------- |
-| `--environment` | `-e`  | Target environment for deployment     | `development`     |
-| `--version`     | `-v`  | Version tag for the deployment        | Current timestamp |
-| `--api-key`     | `-k`  | API key for authentication (required) | None              |
+#### Required Options
+
+| Option          | Alias | Description                    | Environment Variable |
+| --------------- | ----- | ------------------------------ | -------------------- |
+| `--api-key`     | `-k`  | API key for authentication     | `MOTIA_API_KEY`      |
+| `--version-name`| `-v`  | Version tag for the deployment | None                 |
+
+#### Optional Options
+
+| Option             | Alias | Description                            | Environment Variable     |
+| ------------------ | ----- | -------------------------------------- | ------------------------ |
+| `--project-id`     | `-p`  | Project ID                             | `MOTIA_PROJECT_ID`       |
+| `--environment-id` | `-s`  | Environment ID                         | `MOTIA_ENVIRONMENT_ID`   |
+| `--env-file`       | `-e`  | Path to environment file               | None                     |
```

Comment:
can you elaborate more about this arg? users may not understand how to add environment variables to their deployments, we need a section explaining it more explicitly